rules/windows/builtin/win_cobaltstrike_service_installs.yml

```diff
@@ -1,25 +1,33 @@
 title: CobaltStrike Service Installations
 id: 5a105d34-05fc-401e-8553-272b45c1522d
-description: Detects known malicious service installs that appear in cases in which a Cobalt Strike beacon elevates privileges
-author: Florian Roth
+description: Detects known malicious service installs that appear in cases in which a Cobalt Strike beacon elevates privileges or lateral movement
+author: Florian Roth, Wojciech Lesicki
 references:
     - https://www.sans.org/webcasts/119395
-date: 2021/05/26
+date: 2021/06/01
```

Comment:
i think you should add a line with modified like that `modified :2021/06/01` under the date rather than modify the date

Reply:
Correct

Reply:
StevenDias33 - thanks, good point.
@Neo23x0 thanks for the commit.